### PR TITLE
Fixes #52 : Added Help Command to CLI

### LINKED ISSUE #52: Add help command to the CLI
Add help command with the instructions about how to use our CLI

### PR DESCRIPTION
**Pull request by ish-u**
**CHANGELOG** :memo:

* added help command

**LABELS**

* Fixes #52

* [Travis CI](https://travis-ci.org/github/ish-u/spacecraft-cli/builds/734581009)

### CODE REVIEW

**Comment by pyymenta** [comment]
Thanks @ish-u! I'll review it soon!

File: src/cli.ts [hunk 1]
```diff
@@ -77,6 +77,28 @@ export default class CLI {
...
+Once Launched, the Ship moves Forward 
+
+W -> Increase Forward Speed
+S -> Decrease Forward Speed 
+A -> Move Left
+D -> Move Right
+Max Speed - 5 speedunit
+Min Speed - 1 speedunit (after leaving (0, 0))
+
+Press H -> Show the Ship Reference Manual
```
Comment (pyymenta) [inline src/cli.ts]: LGTM :rocket: 

@ish-u  Could you add a message [here](https://github.com/pyymenta/spacecraft-cli/blob/master/src/cli.ts#L86) with the instructions to access the help screen?

Comment (ish-u) [reply src/cli.ts]: Ok i will do that today

Comment (ish-u) [reply src/cli.ts]: I have added the instruction to access Help Screen.

**Comment by sudiptog81** [comment]
There were some merge conflicts where I had to help @ish-u, we should mention in `CONTRIBUTING.md` for contributors to work on their own branch and keep `master` in sync with `upstream`. You can squash and merge this PR keeping contents of this PR.

**Comment by pyymenta** [comment]
> There were some merge conflicts where I had to help @ish-u, we should mention in `CONTRIBUTING.md` for contributors to work on their own branch and keep `master` in sync with `upstream`. You can squash and merge this PR keeping contents of this PR.

Thanks for your suggestion, I'll add this information in `CONTRIBUTING.md`. Although we are unable to merge this PR because we still have some conflicts that we need to fix.

**Comment by ish-u** [comment]
@pyymenta 
Please merge #64 to fix #52 as it doesn't have any conflicts and close this PR.
Sorry if my PR caused any trouble.

**Comment by pyymenta** [comment]
> @pyymenta
> Please merge #64 to fix #52 as it doesn't have any conflicts and close this PR.
> Sorry if my PR caused any trouble.

No problem man, feel free to take [another challenge](https://github.com/pyymenta/spacecraft-cli/issues)! Again, thanks for your contribuition! :1st_place_medal: :1st_place_medal:

**Pull request closed without merging**